### PR TITLE
Updated the document

### PR DESCRIPTION
**Pull request by joinimran**
Updated the steps in the document related to Windows Analytics Solutions.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4392

### CODE REVIEW

**Comment by jvsam** [comment]
@nenonix and @JohanFreelancer9 for copy editing. Thank you.

**Comment by jvsam** [comment]
Sorry @nenonix @JohanFreelancer9 please hold. @joinimran pleawse verify the steps, shouldn't it be Log Analytics workspaces > Solutions and not Log Analytics > Solutions? Please check again, thank you.

File: windows/deployment/update/windows-analytics-FAQ-troubleshooting.md [hunk 1]
```diff
@@ -53,12 +55,12 @@ If you've followed the steps in the [Enrolling devices in Windows Analytics](win
...
+In Log Analytics, go to **Solutions** and verify that you are subscribed to the Windows Analytics solutions you intend to use.
 
 Even though devices can take 2-3 days after enrollment to show up due to latency in the system, you can now verify the status of your devices within a few hours of running the deployment script as described in [You can now check on the status of your computers within hours of running the deployment script](https://techcommunity.microsoft.com/t5/Windows-Analytics-Blog/You-can-now-check-on-the-status-of-your-computers-within-hours/ba-p/187213) on the Tech Community Blog.
 
 >[!NOTE]
-> If you generate the status report and get an error message saying "Sorry! We’re not recognizing your Commercial Id," go to **Settings > Connected sources > Windows telemetry** remove the Upgrade Readiness solution, and then re-add it.
+> If you generate the status report and get an error message saying "Sorry! We’re not recognizing your Commercial Id, See [Enrolling devices in Windows Analytics.](https://docs.microsoft.com/windows/deployment/update/windows-analytics-get-started)
```
Comment (illfated) [inline windows/deployment/update/windows-analytics-FAQ-troubleshooting.md]: Cosmetic detail: it is normal to put any punctuation mark (period, colon or comma) on the outside of link texts, especially if the punctuation ends a sentence or a sentence clause.
```suggestion
> If you generate the status report and get an error message saying "Sorry! We’re not recognizing your Commercial Id, See [Enrolling devices in Windows Analytics](https://docs.microsoft.com/windows/deployment/update/windows-analytics-get-started).
```

**Comment by joinimran** [comment]
@jvsam , Its a Log Analytics troubleshooting document and not a Log Analytics Setup procedure, So my understanding is that when you are troubleshooting any of the components, you are already there in this case you are already in Log Analytics and proceeding further. The document you referred is a procedure to setup Log Analytics. But if you are still on it. Let me know, I  will add these steps as well. But from my point of view, these steps are not required.

File: windows/deployment/update/windows-analytics-FAQ-troubleshooting.md [hunk 2]
```diff
@@ -230,9 +232,7 @@ We have identified an incompatibility between AbnormalShutdownCount and the Limi
 
 If you want to stop using Upgrade Readiness and stop sending diagnostic data to Microsoft, follow these steps:
 
-1. Unsubscribe from the Upgrade Readiness solution in Azure Portal. In Azure Portal, go to **Settings** > **Connected Sources** > **Windows Telemetry** and choose the **Unsubscribe** option.
-
-   ![Upgrade Readiness unsubscribe](images/upgrade-analytics-unsubscribe.png)
+1. Delete the Upgrade Readiness Solution in Log Analytics Workspace. In Log Analytics Workspace go to, **Solutions > Compatibility Assessment > Delete**
```
Comment (nenonix) [inline windows/deployment/update/windows-analytics-FAQ-troubleshooting.md]:
```suggestion
1. Delete the Upgrade Readiness solution in Log Analytics workspace. In Log Analytics workspace. select **Solutions** > **Compatibility Assessment** > **Delete**.
```

**Comment by joinimran** [comment]
Ready to go live.

**Comment by jvsam** [comment]
@joinimran actually it's more about the terminology. In the Azure portal, it does not show it as "Log Analytics' rather it's 'Log Analytics workspace' so the steps need to be accurate to avoid confusions, which what prompted this request from the community user in the first place,  because the steps are not the same as what he is seeing in the portal. Unless you are seeing it differently on your end? 

[image](https://user-images.githubusercontent.com/18371314/63111754-38e8b680-bf43-11e9-8dd2-a325975f9498.png)

This is via the Azure portal:

image

image


CC: @nenonix

**Comment by joinimran** [comment]
@jvsam Ok, I got the missing point in the whole conversation. It was Log Analytics workspace on the top and I have updated the document accordingly. 

Regards
Imran

**Comment by jvsam** [comment]
Hi @greg-lindsay this pull request has been copy edited and is ready for your final review and approval. It contains correct steps on how to access and delete Windows Analytics solutions in the Azure portal (via Azure Log Analytics workspace). See: commits and [changes](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4687/files). 

Please let us know if there are outstanding issues to be resolved prior to merge. Thank you very much.

**Comment by jvsam** [comment]
Hello again @greg-lindsay, if you have time, can you please review this PR and merge if there are no issues? This contains the correct instructions in navigating through the Azure portal. 

@AndreaBarr can you please check this as well? Thank you.

**Comment by AndreaBarr** [comment]
@greg-lindsay please review this for merging

**Comment by jvsam** [comment]
Hi @greg-lindsay just following up. Would you be able to review this pull request? Thank you.

CC: @AndreaBarr

**Comment by illfated** [comment]
This time I would like to note that it is equally interesting to view the changes in default diff view
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4687/files
as it is to view the rich text diff:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4687/files?short_path=bf94dd8#diff-bf94dd8df72d76460bf732f97495fa92

**Comment by jvsam** [comment]
Hello again @greg-lindsay, hoping you'll have time soon to look into this PR and review the proposed changes that will resolve #4392. Thank you.

CC: @AndreaBarr

**Comment by jvsam** [comment]
Hello again @greg-lindsay just following up regarding the status of this pull request. We're hoping you'll have time to review this soon. Thank you.

**Comment by greg-lindsay** [comment]
I need to switch the writer on this topic back to Jaime Ondrusek. I'm not the writer here, but I'll approve the change.

**Comment by jvsam** [comment]
Thank you very much for merging this PR @greg-lindsay. Noted on the doc author/writer. CC: @jaimeo

**Pull request merged**